airflow/www/views.py

```diff
@@ -2232,57 +2232,6 @@ def dagrun_queued(self):
         confirmed = request.form.get('confirmed') == 'true'
         return self._mark_dagrun_state_as_queued(dag_id, dag_run_id, confirmed)
 
-    @expose("/dagrun_details")
```

Comment:
To avoid breaking links, we should make this redirect to the appropriate spot in the grid view.

Reply:
Nice! Will do.

Reply:
This is fixed now :). 
Tested it and works well.
I removed the unnecessary `@action_logging` and `@access` decorators as well.